web/README.md

```diff
@@ -26,8 +44,8 @@ yarn build
 
 The resulting `build` directory can be served using any static hosting service.
 
-We use a [github actions workflow][workflow] to deploy this site with every push
-to master.
+For the main site, we use a [github actions workflow][workflow] to deploy with
+every push to master.
 
 [site]: https://vast.io/
 [workflow]: https://github.com/tenzir/vast/blob/master/.github/workflows/docs.yaml
```

Comment:
This link is outdated; we build as part of the VAST workflow instead, and deploy in a separate deployment environment: https://github.com/tenzir/vast/actions/workflows/pages/pages-build-deployment